test/functional/Jsr292/src/com/ibm/j9/jsr292/api/MethodHandleAPI_arrayConstructor.java

```diff
@@ -180,6 +180,20 @@ public static void test_arrayConstructor_StringArray() throws Throwable {
 		Assert.assertEquals(3, stringArray.length);
 	}
 	
+	/**
+	 * arrayConstructor test for a StringBuffer array
+	 * @throws Throwable
+	 */
+	@Test(groups = { "level.extended" }, invocationCount = 2)
+	public static void test_arrayConstructor_StringBufferArray() throws Throwable {
+		MethodHandle arrayLength = MethodHandles.arrayLength(StringBuffer[].class);
+		MethodHandle mhArrayConstructor = MethodHandles.arrayConstructor(StringBuffer[].class);
+		Assert.assertEquals(mhArrayConstructor.type(), methodType(StringBuffer[].class, int.class));
+		
+		Object stringBufferArray = mhArrayConstructor.invoke(10);
+		Assert.assertEquals(arrayLength.invoke(stringBufferArray), 10);
```

Comment:
Please use `invokeExact` rather than `invoke` as it ensures we've got the types exactly as expected.

Reply:
Already updated in test cases.